src/preferences/dialog/dlgprefmetadatadlg.ui

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>DlgPrefMetadataDlg</class>
+ <widget class="QWidget" name="DlgPrefMetadataDlg">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>668</width>
+    <height>514</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Metadata Broadcast Preferences</string>
```

Comment:
```suggestion
   <string>Metadata Output Preferences</string>
```
I think the conflation of these new features with broadcasting would confuse a lot of users.